pkg/stub/handler.go

```diff
@@ -1194,11 +1303,27 @@ type defaultSDKWrapper struct {
 }
 
 func (g *defaultSDKWrapper) Update(samplesResource *v1alpha1.SamplesResource) error {
-	return sdk.Update(samplesResource)
+	err := sdk.Update(samplesResource)
+	count := 0
+	for count < 10 {
+		if !g.h.IsRetryableAPIError(err) {
+			break
+		}
+		err = sdk.Update(samplesResource)
+	}
```

Comment:
should probably have a delay here.. use the wait.Poll helper.

Reply:
Yeah forgot about that (used it elsewhere) ... thanks